de.metas.business/src/main/java/de/metas/pricing/DiscountSchemaId.java

```diff
@@ -0,0 +1,64 @@
+package de.metas.pricing;
```

Comment:
why we need this?
PricingConditionsId is not sufficient?

Reply:
note: after replacing DiscountSchemaId with PricingConditionsId... pay attention to variable/parameter names.
i.e. `discountSchemaId` is not legit in our terminology

Reply:
Done. Also renamed.